server/mailrepository/mailrepository-cassandra/src/main/java/org/apache/james/mailrepository/cassandra/CassandraMailRepositoryMailDAO.java

```diff
@@ -44,13 +44,9 @@
 import static org.apache.james.mailrepository.cassandra.MailRepositoryTable.SENDER;
```

Comment:
I am :+1: with this change, however, I would like to agree on the fact that this is a breaking change.

Upgrade procedure: For each Cassandra repository, before the upgrade
 - Create a file repository
 - Create a mailet processor storing emails in that file mail repository
 - Reprocess data of the Cassandra mail repository to the file one

Then perform the upgrade with an empty Cassandra repository. Once done:
 - Create a mailet processor storing emails back in the Cassandra mail repository
 - Reprocess all emails
 - Delete the temporary file mail repository

Seems painful...